civis/ml/_model.py

```diff
@@ -740,6 +741,7 @@ def from_existing(cls, train_job_id, train_run_id='latest', client=None):
                     git_token_name=git_token_name,
                     verbose=args.get('DEBUG', False))
         klass.train_result_ = fut
+        klass._etl_train = args.get('ETL', None)
```

Comment:
Isn't `_etl_train` an `Estimator` when it gets set in `train`? Does it make sense to have it mean something different here? FWIW, it looks like we aren't setting `self._input_model` here.